packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1346,6 +1346,18 @@ export const audiusBackend = ({
     }
   }
 
+  async function getEntityManagerReplicaSetEnabled() {
+    try {
+      const res = await fetch(
+        `${identityServiceUrl}/health_check/entity_manager_replica_set_enabled`
+      ).then((res) => res.json())
```

Comment:
generally better practice to not mix async await and .then syntax
I probably wrote the method above this tho
```
const res = await fetch(url)
const resJson = await res.json()
return res.entity...
```

Reply:
ooh true ill address both